Mathlib/Analysis/Meromorphic/TrailingCoefficient.lean

```diff
@@ -141,6 +141,34 @@ lemma MeromorphicAt.meromorphicTrailingCoeffAt_ne_zero (h₁ : MeromorphicAt f x
   obtain ⟨g, h₁g, h₂g, h₃g⟩ := (meromorphicOrderAt_ne_top_iff h₁).1 h₂
   simpa [h₁g.meromorphicTrailingCoeffAt_of_ne_zero_of_eq_nhdsNE h₂g h₃g] using h₂g
 
+/--
+The trailing coefficient of a constant function is the constant.
+-/
+theorem meromorphicTrailingCoeffAt_const {x : 𝕜} {e : 𝕜} :
+    meromorphicTrailingCoeffAt (fun _ ↦ e) x = e := by
+  by_cases he : e = 0
+  · rw [he]
+    apply MeromorphicAt.meromorphicTrailingCoeffAt_of_order_eq_top
+    rw [meromorphicOrderAt_eq_top_iff]
+    simp
+  · exact analyticAt_const.meromorphicTrailingCoeffAt_of_ne_zero he
+
+open Classical in
+/--
+The trailing coefficient of `fun z ↦ z - constant` at `z₀` equals one if `z₀ = constant`, or else
+`z₀ - constant`.
+-/
+theorem meromorphicTrailingCoeffAt_id_sub_const {x y : 𝕜} :
+    meromorphicTrailingCoeffAt (· - y) x = if x = y then 1 else x - y := by
+  by_cases h : x = y
+  · simp_all only [sub_self, ite_true]
+    apply AnalyticAt.meromorphicTrailingCoeffAt_of_ne_zero_of_eq_nhdsNE (n := 1) (by fun_prop)
+      (by apply one_ne_zero)
+    simp_all
```

Comment:
```suggestion
    simp
```
I know you love `simp_all`, but when a bare `simp` it's better (in the sense it's more robust, so it will require less maintenance later on)